Google.Authenticator/TwoFactorAuthenticator.cs

```diff
@@ -171,16 +187,43 @@ public string GetCurrentPIN(string accountSecretKey)
             return GeneratePINAtInterval(accountSecretKey, GetCurrentCounter());
         }
         
-         public string GetCurrentPIN(string accountSecretKey,DateTime now)
+        public string GetCurrentPIN(string accountSecretKey, DateTime now)
         {
             return GeneratePINAtInterval(accountSecretKey, GetCurrentCounter(now,_epoch,30));
         }
 
+        public string[] GetCurrentPINs(byte[] accountSecretKey)
+        {
+            return GetCurrentPINs(accountSecretKey, DefaultClockDriftTolerance);
+        }
+
         public string[] GetCurrentPINs(string accountSecretKey)

```

Comment:
I feel like this should now be changed to call the new `GetCurrentPINs(byte[]...` - it feels like it's the right thing to do. 
In essence I think I am asking you to extend your PR a tad to make byte arrays the default for the secret and convert the existing methods that take strings to just be convenience wrappers. 
Am I overthinking this? :)

Reply:
I don't think you're overthinking it. When I looked at it I felt like that needed to happen as well.